apps/jupyter/extensions/notebook-runner/src/task-importer.ts

```diff
@@ -0,0 +1,71 @@
+import JSZip from "jszip";
+import { CrtInternalFiles } from "./task-format";
+
+export interface CrtInternalTask {
+  taskTemplate: Blob;
+  studentTask: Blob;
+  autograder: Blob;
+  data: Map<string, Blob>;
+  gradingData: Map<string, Blob>;
+  src: Map<string, Blob>;
+  gradingSrc: Map<string, Blob>;
+}
+
+export interface ExternalCustomTask {
+  taskFile: Blob;
+  data: Map<string, Blob>;
+  gradingData: Map<string, Blob>;
+  src: Map<string, Blob>;
+  gradingSrc: Map<string, Blob>;
+}
+
+const extractFolder = async (
+  zip: JSZip,
+  prefix: string,
+): Promise<Map<string, Blob>> => {
+  const files = new Map<string, Blob>();
+
+  const folderPath = prefix.endsWith("/") ? prefix : `${prefix}/`;
+
+  for (const [path, file] of Object.entries(zip.files)) {
+    const relativePath = path.substring(folderPath.length);
+    if (relativePath) {
+      const blob = await file.async("blob");
+      files.set(relativePath, blob);
+      console.log(`extracted from ${relativePath} from ${folderPath}`);
```

Comment:
Is the log here left intentionally? If so, may I suggest using `console.debug` instead?